mailet/standard/src/main/java/org/apache/james/transport/matchers/dlp/DlpDomainRules.java

```diff
@@ -0,0 +1,203 @@
+/****************************************************************
```

Comment:
This class is rather long... I'd extract some part of it in another class

Reply:
I don't agree it's long. There's a lot of convenience methods but if you look at real code, there's around 20 lines of logic.

Reply:
I don't agree it's long. There's a lot of convenience methods but around 20 lines of logic